src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -437,50 +470,7 @@ public async Task ResponseStreamFrames_HeadersAfterHeadersAndContinuationWithout
                 await server.WriteFrameAsync(MakeSimpleHeadersFrame(streamId, endHeaders: false));
 
                 // As this is a connection level error, the client should see the request fail.
-                await Assert.ThrowsAsync<HttpRequestException>(async () => await sendTask);
-
-                // The client should close the connection as this is a fatal connection level error.
-                Assert.Null(await server.ReadFrameAsync(TimeSpan.FromSeconds(30)));
-            }
-        }
-
-        [ConditionalFact(nameof(SupportsAlpn))]
-        public async Task ResponseStreamFrames_HeadersAfterHeadersWithEndHeaders_ConnectionError()
-        {
-            using (var server = Http2LoopbackServer.CreateServer())
-            using (var client = CreateHttpClient())
-            {
-                Task sendTask = client.GetAsync(server.Address);
-                await server.EstablishConnectionAsync();
-                int streamId = await server.ReadRequestHeaderAsync();
-
-                await server.WriteFrameAsync(MakeSimpleHeadersFrame(streamId, endHeaders: true));
-                await server.WriteFrameAsync(MakeSimpleHeadersFrame(streamId, endHeaders: false));
-
-                // As this is a connection level error, the client should see the request fail.
-                await Assert.ThrowsAsync<HttpRequestException>(async () => await sendTask);
-
-                // The client should close the connection as this is a fatal connection level error.
-                Assert.Null(await server.ReadFrameAsync(TimeSpan.FromSeconds(30)));
-            }
-        }
-
-        [ConditionalFact(nameof(SupportsAlpn))]
-        public async Task ResponseStreamFrames_HeadersAfterHeadersAndContinuationWithEndHeaders_ConnectionError()
-        {
-            using (var server = Http2LoopbackServer.CreateServer())
-            using (var client = CreateHttpClient())
-            {
-                Task sendTask = client.GetAsync(server.Address);
-                await server.EstablishConnectionAsync();
-                int streamId = await server.ReadRequestHeaderAsync();
-
-                await server.WriteFrameAsync(MakeSimpleHeadersFrame(streamId, endHeaders: false));
-                await server.WriteFrameAsync(MakeSimpleContinuationFrame(streamId, endHeaders: true));
-                await server.WriteFrameAsync(MakeSimpleHeadersFrame(streamId, endHeaders: false));
-
-                // As this is a connection level error, the client should see the request fail.
-                await Assert.ThrowsAsync<HttpRequestException>(async () => await sendTask);
```

Comment:
I deleted these two tests because, with the addition of trailing headers support, they were no longer testing what they were intended to test.  With trailing headers support, it's valid to send headers after an END_HEADER, but they're interpreted as trailers.  I instead added a test to the trailers test to validate the behavior when no data is sent between headers and trailing headers.